test/extended/fixtures/forcepull-false-cust.json

```diff
@@ -0,0 +1,384 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1beta3",
+  "metadata": {
+    "name": "ruby-helloworld-sample",
+    "creationTimestamp": null,
+    "annotations": {
+      "description": "This example shows how to create a simple ruby application in openshift origin v3"
+    }
+  },
+  "objects": [
+    {
+      "kind": "Service",
```

Comment:
you can strip these templates down to the bare bones...  you shouldn't need anything more than the BuildConfig, as long as you use a DockerImage as the From type for the builder image.